PythonRegression/util/static_vals.py

```diff
@@ -27,4 +29,3 @@
 P9DM9ME9TO9STOP999999999PN9Q99LKE999999999MMMMMMMMMHEA999999999E99999999999999"
 
```

Comment:
Move to Example tables in Scenario Outlines

Reply:
Moving the full hash and trytes to the example table won't work because you cannot nest a long string using ` ``` ` from within the table. This would leave the strings too long to view on github, or any IDE really. I did convert it to a scenario outline that takes in the input variable name for the `static_vals.py` variable that should be compared.